### ISSUE TITLE
Add configuration for bonds on interfaces

### ISSUE THREAD

**Issue by usamaahmadkhan** [redhat-cop/ocp4-helpernode#295]
Bonding is a very typical scenario in production systems.
Bond configurations can be passed similarly to ip configurations as described in the docs here: [bonding configurations](https://docs.openshift.com/container-platform/4.10/installing/installing_bare_metal/installing-bare-metal.html#installation-user-infra-machines-routing-bonding_installing-bare-metal) 

For both:
- masters
- workers

Happy to open a PR if this issue gets accepted.

**Comment by christianh814** [comment]
Yeah, if you have a bond you want to set up, that makes sense to want to configure the helpernode with that.

I would just make it a conditional. As long as you keep the default setup and people can opt in for bonding, I'm all for it
